src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/rca/OldGenRca.java

```diff
@@ -72,7 +73,7 @@ protected double getMaxOldGenSizeOrDefault(final double defaultValue) {
       }
       double ret =
           SQLParsingUtil
-              .readDataFromSqlResult(heapMaxMetric.getData(), MEM_TYPE.getField(), OLD_GEN.toString(), MetricsDB.MAX);
+              .readDataFromSqlResult(heapMaxMetric.getData(), MEM_TYPE.getField(), HEAP.toString(), MetricsDB.MAX);
```

Comment:
I see that you have updated OLD_GEN to HEAP. The method says getMaxOldGenSizeOrDefault. Is there a reason why this change was made? Do we want to rename the method?

Reply:
yes, this was a bug I found in old gen RCA. Heap usage is supposed to be calculated by (old gen usage)/(max total heap size). but for some reason, we calculated it by (old gen usage)/(max old gen size)

Reply:
done. added a new function to retrieve HEAP and leave the OLD_GEN unchanged